assets/js/modules/analytics/components/dashboard/DashboardAllTrafficWidgetV2/UserDimensionsPieChart.js

```diff
@@ -84,6 +85,10 @@ export default function UserDimensionsPieChart( { dimensionName, entityURL, sour
 		const { chart, onSelect } = chartData || {};
 		const { slices } = UserDimensionsPieChart.chartOptions;
 
+		if ( selectedRow !== undefined ) {
+			chart.setSelection( [ { row: selectedRow } ] );
```

Comment:
I think the underlying problem here might be that we're selecting the `row` which is not specific enough, and it also shouldn't be necessary. We already store when a pie slice is selected, by storing the `dimensionValue` which _is_ the pie slice.

Basically, _if_ there is a (non-empty) `dimensionValue`, we need to ensure that the `row` which has that value as label (i.e. `row[ 0 ]` if I'm not mistaken) is selected.